README.md

```diff
@@ -303,6 +303,7 @@ Providers
 * League Of Legends
 * Lebowski
 * Lord Of The Rings
+* Locale
```

Comment:
What is better, `Locale` or `LocalePicker`, because in the code, this faker is called `localePicker()`?

Reply:
I think it's better if this list is generated (like you did with the Markdown) and matches the code. However, Locale, while a good name, is a bit conflicting with `java.util.Locale`, so maybe LocaleProvider (or picker), or perhaps Internationalization? (but then again, a lot of the Faker is related to i18n). Tricky question!

Reply:
How about renaming the class name to `LocaleProvider` and using this name throughout the project?

Reply:
Or `LocaleFaker` is better. I see, that in the code there are `Base` and `Provider` suffixes.

Reply:
What about plural `Locales`?
`Base` and `Provider` suffixes are used for abstract classes/interfaces and there is no need for every class add such suffixes. It's clear that if any class extends/implements anything with `Base`/`Provider` then it will definitely be `Base`/`Provider`

Reply:
I thought about Locales too, but it's not like we have `Names`, `Addresses`, or even `Dogs`. Basically, all our providers are singular, so maybe that's not ideal. How about calling it `Locality` or so? I'm no native speaker, but it's a bit of a synonym in most cases, but yes, I know it generates `Locale`s, so that's not ideal either perhaps.

Reply:
i'm ok with `Locality` as well

Reply:
Then, I will rename to `Locality`